persistence-model/src/main/resources/application-dev.conf

```diff
@@ -4,13 +4,26 @@ akka {
     warn-about-java-serializer-usage = on
     serializers {
       institution = "hmda.persistence.serialization.institutions.InstitutionProtobufSerializer"
+      submission = "hmda.persistence.serialization.submission.SubmissionProtobufSerializer"
       filing = "hmda.persistence.serialization.filing.FilingProtobufSerializer"
+      parser = "hmda.persistence.serialization.parser.HmdaFileParserProtobufSerializer"
+      larValidated = "hmda.persistence.serialization.validation.LarValidatedProtobufSerializer"
+      upload = "hmda.persistence.serialization.upload.UploadProtobufSerializer"
     }
     serialization-bindings {
       "hmda.persistence.messages.events.institutions.InstitutionEvents$InstitutionCreated" = institution
       "hmda.persistence.messages.events.institutions.InstitutionEvents$InstitutionModified" = institution
       "hmda.persistence.messages.events.institutions.FilingEvents$FilingCreated" = filing
       "hmda.persistence.messages.events.institutions.FilingEvents$FilingStatusUpdated" = filing
+      "hmda.persistence.messages.events.institutions.SubmissionEvents$SubmissionCreated" = submission
```

Comment:
Yes, it's needed in both places. `application-dev.conf` uses leveldb in `demo` mode, while `application.conf` is meant for production and uses `Cassandra`. To me it makes sense to have both using the same serialization format for consistency